### DISCUSSION TITLE
"watch time" API reference

### DISCUSSION THREAD

**Discussion by maxrave-dev** [sigma67/ytmusicapi#456] [Q&A]
When I research YouTube Music Internal API, I see "watchtime" GET request and it has many random parameters. And "add to history" in ytmusicapi only adds the item to history, but don't make YouTube Music Home Page personalized from the item I added to history. So maybe "watchtime" reference can do it, did you know something about these reference parameters? Thanks

**Comment by sigma67** [comment]
Not sure, what is the endpoint?

GET request does not modify state so it should not have any effect

**Reply by maxrave-dev** [reply]
![image](https://github.com/sigma67/ytmusicapi/assets/113747128/04e3ea5c-dba1-4268-8f9a-bb1071c14916)
Check this image

**Reply by sigma67** [reply]
Hm, I guess if you see a point in implementing it feel free to submit a PR. I never saw a need to add it
